### PR TITLE
Try and fix dask CPU workers

### PR DESCRIPTION
**Pull request by hmacdope**
Adds ability use dask-workers on CPUqueue using suggested fix from MSK HPC.

### CODE REVIEW

**Comment by hmacdope** [comment]
@JenkeScheen confirmed that this works on lilac, and should also fix the issue of core dumps on workers. Just add `ulimit -c 0` in your supervisory job also.

File: asapdiscovery-docking/asapdiscovery/docking/workflows/large_scale_docking.py [hunk 1]
```diff
@@ -124,7 +124,7 @@ class LargeScaleDockingInputs(BaseModel):
     )
 
     dask_cluster_max_workers: PositiveInt = Field(
-        40, description="Maximum number of workers to use for Lilac dask cluster"
+        200, description="Maximum number of workers to use for Lilac dask cluster"
```
Comment (JenkeScheen) [inline asapdiscovery-docking/asapdiscovery/docking/workflows/large_scale_docking.py:127]: why 200?

**Review by JenkeScheen — APPROVED** [review]
LGTM!

File: asapdiscovery-docking/asapdiscovery/docking/workflows/large_scale_docking.py (hunk 1 above)
Comment (hmacdope) [reply asapdiscovery-docking/asapdiscovery/docking/workflows/large_scale_docking.py:127]: More workers = more fun.

**Pull request merged**